zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/identity/RoleRemoveEntityProcessor.java

```diff
@@ -114,6 +118,9 @@ private boolean isEntityPresent(final long entityKey, final EntityType entityTyp
     if (EntityType.USER == entityType) {
       return userState.getUser(entityKey).isPresent();
     }
+    if (EntityType.MAPPING == entityType) {
+      return mappingState.get(entityKey).isPresent();
+    }
```

Comment:
🙃  Might be more readable and maintainable to introduce a `switch` statement here.